examples/jenkins-master/jenkins-master-template.json

```diff
@@ -0,0 +1,251 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins-master",
+    "annotations": {
+      "description": "Jenkins service with Kubernetes plugin, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```

Comment:
so what bothers me about this is the implication that to do master/slave I need to build a new custom jenkins image, which you don't.  it just happens that this particular example needs additional stuff.

it makes it appear overly burdensome to setup a jenkins master/slave config.